lib/v2/ieee/latestvol.js

```diff
@@ -0,0 +1,86 @@
+const got = require('@/utils/got');
+const path = require('path');
+const { art } = require('@/utils/render');
+
+module.exports = async (ctx) => {
+    const punumber = ctx.params.journal;
+    const sortType = ctx.params.sortType || 'vol-only-seq';
```

Comment:
```suggestion
    const sortType = ctx.params.sortType ?? 'vol-only-seq';
```

Reply:
Thanks your attention, I have modified it.